docs/docs/react-without-jsx.md

```diff
@@ -33,16 +35,30 @@ class Hello extends React.Component {
 }
 
 ReactDOM.render(
-  React.createElement(Hello, {toWhat: 'World'}, null),
+  React.createElement(Hello, { toWhat: 'World' }, null),
   document.getElementById('root')
 );
 ```
 
+`React.createElement()` takes three arguments arguments: a component, a `props` object, and any number of children as the final arguments.
+
+The component argument can take a number of forms:
+
+1. A string - `React.createElement("div", null, null)`
+2. A sub-class of `React.Component` - `React.createElement(Hello, null, null)`
+3. A plain stateless-functional componet - `React.createElement(HelloFunctionalComponent, null, null)`
+
+To learn more about the function signature and arguments, you can read [the API documentation.](/docs/react-api.html#createelement)
+
+### Examples 
+
 If you're curious to see more examples of how JSX is converted to JavaScript, you can try out [the online Babel compiler](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-0&code=function%20hello()%20%7B%0A%20%20return%20%3Cdiv%3EHello%20world!%3C%2Fdiv%3E%3B%0A%7D).
 
-The component can either be provided as a string, or as a subclass of `React.Component`, or a plain function for stateless components.
+If you want to use `React.createElement()` API without ES6, follow the [React Without ES6 Guide](/docs/react-without-es6.html) and use `React.createElement()` instead of JSX.
 
-If you get tired of typing `React.createElement` so much, one common pattern is to assign a shorthand:
+### Eliminating Boilerplate
+
+If writing `React.createElement()` often becomes tiring, a common pattern is to assign the function to a shorter variable and use it as shorthand. Using this shorthand allows for creating elements without JSX to be almost as convenient as creating elements with JSX.
```

Comment:
This is a bit hard to read because the sentences repeat a few phrases too closely. shorter/shorthand/shorthand; creating elements / creating elements.

Reply:
I agree, this can be improved. 

I went through a few iterations and came to the following. If there are any improvements you can think of let me know and I can put whatever final version you want in the PR.

> Though using `React.createElement()` instead of JSX offers the same functionality and versatility of JSX, over time it may become cumbersome to write `React.createElement()` so frequently throughout your application. To solve this problem, we can think in JavaScript - simply assign the function to a variable for a shorthand during re-use.
 
```javascript
const e = React.createElement;

ReactDOM.render(
  e('div', null, 'Hello World'),
  document.getElementById('root')
);
```

Reply:
I also think a brief section should be added in this doc about nesting. It seems obvious to someone familiar with React, but to a beginner unfamiliar with function composition, it may not carry over from JSX super easily.

How do you feel about a code example like this?

```javascript
class Hello extends React.Component {
  render() {
    return React.createElement('div', null, React.createElement('p', null, 'Hello, World!'));
  }
}
```

Reply:
- The community seems to have settled on `h()` as a shorthand for `createElement`, so I'd suggest using that
- The nested example is a lot easier to read with the shorthand + a newline:

```jsx
const h = React.createElement

class Hello extends React.Component {
  render() {
    return h('div', {},
      h('p', {}, 'Hello, World!')
    );
  }
}
```